cmd/minishift/cmd/util/util.go

```diff
@@ -97,3 +98,9 @@ func IsValidProfile(profileName string) bool {
 	}
 	return false
 }
+
+// IsValidProfileName return true if profile name follow ^[a-zA-Z]+[0-9]*$
+func IsValidProfileName(profileName string) bool {
+	rr := regexp.MustCompile(`^[a-zA-Z]+[0-9]*$`)
```

Comment:
I thought we generally allow alpha-numeric. What's the reason for enforcing that the profile name starts with letters?

Reply:
you don't like character classes like {:alpha:] and [:alnum:]?

Instead of repeating the regexp in the docs, I rather state the intention in words.

Reply:
Why is '-' or '_' not allowed? It is nice to at least be able to use some form of "separator" to make names more readable. IMO [:word:] is the character class I would match against, without any exception.

Reply:
> Why is '-' or '_' not allowed? It is nice to at least be able to use some form of "separator" to make names more readable. IMO [:word:] is the character class I would match against, without any exception.

Agree. Can we cross check this with libvirt, xhyve and Virtualbox? I do not think HyperV is an issue as it uses uuid for VM name.